### DISCUSSION TITLE
SQL*Plus binary not found in usr/bin

### DISCUSSION THREAD

**Discussion by checkinindza** [dbeaver/dbeaver#23458] [DBeaver Development]
### Description

So I'll start with the fact I'm completely new to everything tied to databases. We just started learning databases this semester, so it might be that I'm just missing something, somewhere. 

Originally, we use SQL Developer to write SQL queries, but as I use Linux, from a quick read, it's a bit of a pain to install it on Linux. DBeaver was available as an AUR package and a few other students recommended it. So I installed that. The issue is, some of the statements are only available in SQL*Plus. I noticed you can execute your script in SQL*Plus, but that's where I'm stuck.

I understood that you have to setup a Local Client to do that, so I followed the[ included guide](https://dbeaver.com/docs/wiki/https://dbeaver.com/docs/dbeaver/Local-Client-Configuration/), but as I understood, you have to have some kind of DB installed. So I installed and configured MariaDB, following [Arch Wiki ](https://wiki.archlinux.org/title/MariaDB). Choose /usr/bin as the home directory for the local client. 

So that was setup and tried the `Execute in SQL*Plus` and I got `SQL*Plus binary not found in /usr/bin` error.

I'm sorry right away if this is not the right place for this. Still figuring out stuff.

### DBeaver Version

Community Edition 24.0.0.202403042157

### Operating System

EndeavourOS with Linux 6.7.9-arch1-1

### Database and driver

JDBC Driver and Oracle database. 

### Steps to reproduce

1. Try to execute a script in SQL*Plus

**Comment by Matvey16** [comment]
@checkinindza Hi. You need an sqlplus binary wich is included in oracle instant client. You can download it from Oracle's official site and it's just a zip archive. You can unzip it and set home directory of the local client to the folder with a client and it should work.
My screenshot is on windows, but it is the same for linux because you don't need to install anything, you just need to download files.
![image](https://github.com/dbeaver/dbeaver/assets/82543000/609bb2af-7273-445d-a84f-8a202ee79908)
